src/mainwindow.cpp

```diff
@@ -367,3 +375,36 @@ void MainWindow::aboutHotspot()
     dialog.adjustSize();
     dialog.exec();
 }
+
+void MainWindow::customContextMenu(const QPoint &point, QTreeView* view, quint32 symbolRole)
+{
+    QModelIndex index = view->indexAt(point);
+    if (index.isValid()) {
```

Comment:
invert and do early-return when it's not valid, decreases the indentation depth below